moped-editor/.env-cmdrc

```diff
@@ -100,13 +100,13 @@
     "REACT_APP_AWS_COGNITO_DOMAIN": "atd-moped-production.auth.us-east-1.amazoncognito.com",
     "REACT_APP_MAPBOX_TOKEN": "pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNraWV4dHR0ZjAwNnYyd3FwYjFoNHduaDcifQ.--3vRm2KHq1gh5K_L0pqtA",
     "REACT_APP_NEARMAP_TOKEN": "NWE1ZTA1YTYtOTg4Yy00ZGNhLWFjMzYtMmYxOTI2M2UyODRk",
-    "REACT_APP_KNACK_DATA_TRACKER_SCENE": 0,
-    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_VIEW": 0,
-    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_VIEW": 0,
-    "REACT_APP_KNACK_DATA_TRACKER_APP_ID": "0123456789ABC9876543210",
-    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_ID_FIELD": "field_TBD",
-    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_NAME_FIELD": "field_TBD",
-    "REACT_APP_KNACK_DATA_TRACKER_CURRENT_STATUS_FIELD": "field_TBD",
-    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_CONNECTION": "field_TBD"
+    "REACT_APP_KNACK_DATA_TRACKER_SCENE": 514,
+    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_VIEW": 3089,
+    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_VIEW": 1483,
+    "REACT_APP_KNACK_DATA_TRACKER_APP_ID": "5815f29f7f7252cc2ca91c4f",
+    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_ID_FIELD": "field_4133",
+    "REACT_APP_KNACK_DATA_TRACKER_PROJECT_NAME_FIELD": "field_3857",
+    "REACT_APP_KNACK_DATA_TRACKER_CURRENT_STATUS_FIELD": "field_4136",
+    "REACT_APP_KNACK_DATA_TRACKER_SIGNAL_CONNECTION": "field_3861"
```

Comment:
It is not a requirement that these values differ from values defined earlier in the file for non-production deployments.

When cloning a Knack application, field code values are frequently (always?) preserved in the clone. Therefore, fields which were existing prior to the Moped development, such as the signals connection field, will maintain their values across deployments.